src/main/java/org/jenkinsci/plugins/dockerhub/notification/DockerHubWebHook.java

```diff
@@ -248,12 +250,17 @@ public boolean schedule(WebHookPayload payload) {
             if (isParameterized()) {
                 Collection<ParameterValue> defaults = getDefaultParametersValues();
                 for (ParameterValue value : defaults) {
-                    if (!value.getName().equalsIgnoreCase(KEY_REPO_NAME) && !value.getName().equalsIgnoreCase(KEY_DOCKER_HUB_HOST)) {
+                    if (!value.getName().equalsIgnoreCase(KEY_REPO_NAME)
+                        && !value.getName().equalsIgnoreCase(KEY_PUSHER)
+                        && !value.getName().equalsIgnoreCase(KEY_TAG)
+                        && !value.getName().equalsIgnoreCase(KEY_DOCKER_HUB_HOST)) {
                         parameters.add(value);
                     }
                 }
             }
             parameters.add(new StringParameterValue(KEY_REPO_NAME, payload.getRepoName()));
+            parameters.add(new StringParameterValue(KEY_PUSHER, payload.getPusher()));
```

Comment:
Since it is possible for `payload.getPusher()` to return `null` it could break the build since some plugins don't expect `null` values in parameters. So you need to do something similar as is done for `payload.getCallbackHost()`.